Anatoly Shlom/flex/css/flex.css

```diff
@@ -0,0 +1,177 @@
+/* Приводим браузерные стили в порядок */
+@import url('normalize.css');
+/* Добавляем необходимый шрифт*/
+@import url('https://fonts.googleapis.com/css?family=Lato:300,400,700');
+* {
+  font-family: 'Lato', sans-serif;
+  font-weight: 300;
+  color: #ffffff;
+  padding: 0;
+  margin: 0;
+}
+/*Добавляем фон и немножко кросбраузерности)*/
+html {
+  background: url("../img/bg.png") no-repeat center center fixed;
+  -webkit-background-size: cover;
+  -moz-background-size: cover;
+  -o-background-size: cover;
+  background-size: cover;
+}
+/*оформляем сам виджет*/
+.widget {
+  background: no-repeat;
+  background-image: url("../img/main_bg.png");
+  background-size: cover;
+  margin: 7% 11% 7% 11%;
+  padding: 10% 0 0 0;
+}
+/*верхняя часть виджета*/
+@media (max-width: 600px) {
+  .widget-data {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+  }
+}
+@media only screen and (min-width: 600px) {
+  .widget-data {
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-end;
+    align-items: flex-start;
+    margin: 0 10% 0 0;
+  }
+}
+/*секция с текущей датой*/
+@media only screen and (min-width: 600px) {
+  .current-date {
+    margin: 20px 20px 0 0 ;
+  }
+}
+.current-date__week-day {
+  font-size: 30px;
+}
+.current-date__day {
+  font-size: 14px;
+}
+/*секция с температурой,локацией,ветром*/
+.weather-block {
+  display: table-cell;
+  text-align: center;
+}
+.weather-block__temperature {
+  font-size: 92px;
+  font-weight: 700;
+}
+.weather-block__location {
+  font-size: 22px;
+}
+.wind-block {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: flex-end;
+}
+.wind-block__speed {
+  font-size: 29px;
+  font-weight: 400;
+}
+.wind-block__speed-format {
+  font-size: 14px;
+  font-weight: 400;
+  padding: 0 0 3px 0;
+  /*три пикселя выше нужны что бы стояли на одной линии значение и единицы измерения
+    Верю в то что это баг шрифта, а не мои кривые руки)*/
+}
+/* Оформляем навбар*/
+.nav-block {
+  background: rgba(0, 0, 0, 0.5);
+  margin: 10% 0 0 0;
+}
+@media (max-width: 600px) {
```

Comment:
why you didn't use `screen` here?

Reply:
I thought i don't have to, because this conditions influence only mobile version